packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -0,0 +1,306 @@
+# Mapping Table Column
+
+## Overview
+
+The `nimble-table-column-mapping` is a component that supports rendering a specific number, boolean, or string value as mapped, predefined, html fragments (with content restrictions).
+
+### Background
+
+[Icon column type issue](https://github.com/ni/nimble/issues/1013)
+
+[Boolean column type issue](https://github.com/ni/nimble/issues/1103)
+
+### Features
+
+-   Supported input:
+    -   string
+    -   number
+    -   boolean
+-   Supported output:
+    -   Text
+    -   One or more Nimble icons (with colors via icon severity property)
+    -   Nimble spinner
+    -   Mixed icons and text
```

Comment:
Will nimble automatically space multiple items, such as icon + text? Or, do we need to make sure to have documentation on the recommended spacing between those items?

Reply:
I think our component will have to do the spacing for them. I've added a remark to this effect. 

The client can't set any styling that would affect the elements we render in the cell view's shadow DOM. (My proposed implementation renders them using the `template`'s `innerHTML`. We're not slotting elements that exist in the light DOM.)